server/container/core/src/test/java/org/apache/james/server/core/MimeMessageWrapperTest.java

```diff
@@ -256,15 +256,15 @@ public void testSizeModifiedHeaders() throws MessagingException {
     public void testSizeModifiedBodyWithoutSave() throws MessagingException {
         String newBody = "This is the new body of the message";
         mw.setText(newBody);
-        assertThat(mw.getSize()).isEqualTo(body.length());
+        assertThat(mw.getSize()).isEqualTo(-1);
     }
 
     @Test
     public void testSizeModifiedBodyWithSave() throws MessagingException {
         String newBody = "This is the new body of the message";
         mw.setText(newBody);
         mw.saveChanges();
-        assertThat(mw.getSize()).isEqualTo(body.length());
+        assertThat(mw.getSize()).isEqualTo(-1);
```

Comment:
Are those test names still making sense with that -1 returned?

Reply:
I think as they don't tell anything about what behaviour is expected but just what part of what is tested...